src/test/java/uk/gov/hmcts/ccd/data/caseaccess/AMSwitchTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.hmcts.ccd.data.caseaccess;
+
+import com.google.common.collect.Lists;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import uk.gov.hmcts.ccd.ApplicationParams;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+
+class AMSwitchTest {
+
+    @Mock
+    private ApplicationParams applicationParams;
+
+    @InjectMocks
+    private AMSwitch amSwitch;
+
+    private static final String DIVORCE_CT = "DIVORCE";
+    private static final String PROBATE_CT = "PROBATE";
+    private static final String CMC_CT = "CMC";
+    private static final String FR_CT = "FR";
+    private static final String TEST_CT = "TEST";
+    private List<String> ccdOnlyWriteCaseTypes = Lists.newArrayList(DIVORCE_CT, CMC_CT, TEST_CT);
+    private List<String> amOnlyWriteCaseTypes = Lists.newArrayList(PROBATE_CT);
+    private List<String> bothWriteCaseTypes = Lists.newArrayList(FR_CT);
+    private List<String> ccdOnlyReadCaseTypes = Lists.newArrayList(CMC_CT, PROBATE_CT);
+    private List<String> amOnlyReadCaseTypes = Lists.newArrayList(DIVORCE_CT, FR_CT, TEST_CT);
+
```

Comment:
Perhaps a test to check that if a case type is not specified in the application.properties then it defaults to `ccd` read/write access

Reply:
Good point. Addressed now.